angular/src/directives/navigation/ion-router-outlet.ts

```diff
@@ -241,6 +241,20 @@ export class IonRouterOutlet implements OnDestroy, OnInit {
     return active ? active.url : undefined;
   }
 
+  /**
+   * Returns the RouteView of the active page of each stack.
+   */
+  getLastRouteView(stackId?: string): RouteView | undefined {
```

Comment:
Is this meant to be part of the public API? If not, can we tag it with `@private` so it doesn't come up in completion?

Reply:
I have marked this as `@internal` similar to other framework methods. I can't mark it as `private` as it is used by external classes [ion-tabs.ts](https://github.com/ionic-team/ionic/pull/18493/files#diff-891ce68da9d3204cb1a1870a225e5350R102)

Reply:
sounds good! Thanks for making the change